From 535c3f96bccea84ae9a0a74f7e77d926c13fd4df Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 27 Dec 2017 22:26:44 +0800
Subject: [PATCH 3148/5242] MLK-17314-3 arm: dts: imx7ulp: update nmi irq
 number

commit  4bf149af5e072ec89dafd8455a559908d45b3a9f from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX7ULP B0 chip, NMI irq number is changed,
update it to make VLLS/VLPS work.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 201ff09..1d78563 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -187,7 +187,7 @@
 		nmi: nmi@40220000 {
 			compatible = "fsl,imx7ulp-nmi";
 			reg = <0x40220000 0x1000>;
-			interrupts = <GIC_SPI 74 IRQ_TYPE_LEVEL_HIGH>;
+			interrupts = <GIC_SPI 90 IRQ_TYPE_LEVEL_HIGH>;
 			status = "okay";
 		};
 
-- 
1.7.9.5

